app/src/pages/inside/logsPage/sauceLabsSection/videoSection/videoPlayer/videoPlayer.jsx

```diff
@@ -23,6 +25,7 @@ export class VideoPlayer extends React.Component {
 
   componentDidMount() {
     this.player = videojs(this.videoNode.current, this.props);
+    this.player.on('play', () => this.props.onPlay());
```

Comment:
Why not just `this.player.on('play', this.props.onPlay);`?

Reply:
You are right, closure is unnecessary here.